src/card_operations/card_pair.c

```diff
@@ -387,5 +392,5 @@ uint32_t card_pair_operation(uint8_t card_number,
   }
 
   nfc_deselect_card();
-  return error_status;
```

Comment:
`error_status` not required at all. Please re-review the change

Reply:
removed